<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rank</class>
 <widget class="QWidget" name="Rank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QTabWidget:pane {border-top:0px solid #e8f3f9;background:  transparent; }</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>24小时</string>
      </attribute>
      <layout class="QGridLayout" name="h24Layout">
       <item row="0" column="0">
        <widget class="QtBookList" name="h24BookList">
         <property name="styleSheet">
          <string notr="true">QListWidget {background-color:transparent;}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>7日</string>
      </attribute>
      <layout class="QGridLayout" name="d7Layout">
       <item row="0" column="0">
        <widget class="QtBookList" name="d7BookList">
         <property name="styleSheet">
          <string notr="true">QListWidget {background-color:transparent;}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>30日</string>
      </attribute>
      <layout class="QGridLayout" name="d30Layout">
       <item row="0" column="0">
        <widget class="QtBookList" name="d30BookList">
         <property name="styleSheet">
          <string notr="true">QListWidget {background-color:transparent;}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtBookList</class>
   <extends>QListWidget</extends>
   <header location="global">.qtlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
